### PR TITLE
Use f33 to run linters

### PR DESCRIPTION
**Pull request by tkdchen**
Also fix detected issues:

* W0222(signature-differs)
* W0707(raise-missing-from)

Signed-off-by: Chenxiong Qi <cqi@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [n/a] JSON/YAML configuration changes are updated in the relevant schema
- [n/a] Changes to metadata also update the documentation for the metadata
- [n/a] Pull request has a link to an osbs-docs PR for user documentation updates
- [n/a] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/plugin.py [hunk 1]
```diff
@@ -467,9 +467,9 @@ def run(self, *args, **kwargs):
         else:
             logger.debug("No Dockerfile path has been specified")
 
-        kwargs['buildstep_phase'] = True
-
-        plugins_results = super(BuildStepPluginsRunner, self).run(*args, **kwargs)
+        plugins_results = super(BuildStepPluginsRunner, self).run(
+            keep_going=keep_going, buildstep_phase=True
```
Comment (mkosiarc) [inline atomic_reactor/plugin.py:471]: shouldn't this be buildstep_phase=buildstep_phase ?

File: atomic_reactor/plugin.py [hunk 2]
```diff
@@ -467,9 +467,9 @@ def run(self, *args, **kwargs):
         else:
             logger.debug("No Dockerfile path has been specified")
 
-        kwargs['buildstep_phase'] = True
```
Comment (tkdchen) [inline atomic_reactor/plugin.py:470]: > shouldn't this be buildstep_phase=buildstep_phase ?

@mkosiarc Here.

Comment (mkosiarc) [reply atomic_reactor/plugin.py:470]: I see, what is the reason we are not having this as default in the function?

**Review by MartinBasti — APPROVED** [review]

File: atomic_reactor/plugin.py [hunk 3]
```diff
@@ -455,7 +455,7 @@ def __init__(self, dt, workflow, plugin_conf, *args, **kwargs):
         super(BuildStepPluginsRunner, self).__init__(
             dt, workflow, 'BuildStepPlugin', plugin_conf, *args, **kwargs)
 
-    def run(self, *args, **kwargs):
+    def run(self, keep_going=False, buildstep_phase=False):
```
Comment (MartinBasti) [inline atomic_reactor/plugin.py:458]: Can we have `buildstep_phase=True` here for consistency? It's BuildStepPlugin

**Review by mkosiarc — APPROVED** [review]

File: atomic_reactor/plugin.py (hunk 3 above)
Comment (tkdchen) [reply atomic_reactor/plugin.py]: @MartinBasti @mkosiarc Updated the parameter's default value. PTAL.

Comment (mkosiarc) [reply atomic_reactor/plugin.py]: ack LGTM

**Comment by ben-alkov** [comment]
Can we please pick one of 'e'/'ex'/'exc' and be consistent?

Personally I vote for 'exc'; we don't have to pay for extra characters in identifiers...

**Comment by ben-alkov** [comment]
@MartinBasti; Would we prefer to see 2-3 separate commits in here?

**Comment by tkdchen** [comment]
> Can we please pick one of 'e'/'ex'/'exc' and be consistent?
> 
> Personally I vote for 'exc'; we don't have to pay for extra characters in identifiers...

Historically, I used to write `e` to reference an error or exception (different programming languages call this kind of thing differently). However, while reviewing the Python document about `raise ... from`, I found out `exc` is much expressive in this context than others.

After we decide whichever name to be used, I suggest to update the name through the code to keep consistency.

**Pull request merged**